lib/boundary/issue-trackers/linear.ts

```diff
@@ -146,7 +146,8 @@ export class Linear implements IssueTracker {
 	}
 
 	async createTracker(incident: Incident): Promise<string> {
-		const projectName = `${this.#config.breakingRoomPrefix}${incident.id}`;
+		const priorityPrefix = this.#getPriorityPrefix(incident.priority);
+		const projectName = `${priorityPrefix}${this.#config.breakingRoomPrefix}${incident.id}`;
```

Comment:
Should we do this as well?

```suggestion
		const projectName = `${priorityPrefix}${this.#config.breakingRoomPrefix}${incident.id}: ${incident.title}`;
```

And then remove `description` from projectPayload below.